lib/embed/viewer/file.rb

```diff
@@ -111,6 +107,80 @@ def self.supported_types
       end
 
       private
+      ## 
+      # Creates an embargo message to be displayed, customized for stanford
+      # only embargoed items
+      # @return [String]
+      def embargo_message
+        message = ['Access is restricted', pretty_embargo_date]
+        if @purl_object.stanford_only_unrestricted?
+          message[0] = message[0] + ' to Stanford-affiliated patrons'
```

Comment:
I think `message[0] << ' to Stanford-affiliated patrons'` may be a bit more idiomatic here.